developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -143,7 +143,7 @@ PHP 5.6 (supported)
 
     $ sudo apt-get update
     $ sudo apt-get install mongodb
-    $ sudo apt-get install php5-mongo
+    $ sudo apt-get install php-mongo
```

Comment:
Well, I've installed a fresh ubuntu 16.04 and cant found any ``php5.6-mongo`` package, but ``php5-mongo`` exists .. in 14.04.

Sorry for the disturbing, we learn every day :)